src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,157 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User validUser;
+    private static final String LOGIN_VALID = "abcdef";
+    private static final String LOGIN_0_LENGTH = "";
+    private static final String LOGIN_3_LENGTH = "abc";
+    private static final String LOGIN_5_LENGTH = "abcde";
+    private static final String PASSWORD_0_LENGTH = "";
+    private static final String PASSWORD_3_LENGTH = "pas";
+    private static final String PASSWORD_5_LENGTH = "passw";
+    private static final String PASSWORD_VALID = "passwo";
+    private static final String PASSWORD_8_LENGTH = "password";
+    private static final int USER_AGE_MIN = 18;
+    private static final int USER_AGE_VALID = 25;
+    private static final long USER_ID_VALID = Long.MAX_VALUE;
+    private static final int USER_AGE_NOT_VALID = 16;
+    private static final int USER_AGE_0_LESS = -16;
```

Comment:
Are you testing 17 edge case?
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/1d90396c-081c-433d-81e5-99545fad4a02)